dashboard/src/pages/apps.tsx

```diff
@@ -59,6 +60,20 @@ export default () => {
       return acc
     }, {} as Record<string, Application[]>)
 
+  const countAppsStatus = (state, buildType) => {
```

Comment:
下のように型を明示すると型の恩恵を得られて間違った比較を事前に防止できます
```suggestion
  const countAppsStatus = (state: ApplicationState, buildType: BuildType): number => {
```